### ISSUE TITLE
Error extensions from generated client response

### ISSUE THREAD

**Issue by amalfatti** [graphql-java-generator/graphql-maven-plugin-project#176]
Is there any way to recover `extensions` field from errors returned by a generated client? 
Currently a `GraphQLRequestExecutionException` exception is thrown which reports only the original error message.
It would actually be useful to have the ability to custom handle the graphql error.

For example, my response is something like:

```json
{
  "data": null,
  "errors": [
    {
      "message": "Table is not migrated",
      "extensions": {
        "errorType": "NOT_MIGRATED",
        "table": "tableName"
      }
    }
  ]
}
```

`GraphQLRequestExecutionException` only report message "Table is not migrated".

**Comment by etienne-sf** [comment]
You are right: there is currently no way to retrieve the extension associated to an error.

I'll change that.

Etienne

**Comment by amalfatti** [comment]
Great!
Thank you

**Comment by etienne-sf** [comment]
Released in the 1.18.10 version
